### PR TITLE
configurable spark user shell

### PR DESCRIPTION
**Pull request by lhoss**
New feature as proposed in https://github.com/azavea/ansible-spark/issues/12

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ spark_log_dir: "/var/log/spark"
 spark_run_dir: "/run/spark"
 spark_user: "spark"           # the name of the (OS)user created for spark
 spark_user_groups: []         # Optional list of (OS)groups the new spark user should belong to
+spark_user_shell: "/bin/false"    # the spark user's default shell
```
Comment (hectcastro) [inline defaults/main.yml:13]: Can you also please add a note about this attribute in the `README`?

**Comment by lhoss** [comment]
added: README for the new role variable spark_user_shell

**Comment by lhoss** [comment]
ps: After this PR merged, I'll fire another PR (this time in a separate branch) adding some useful ansible role 'tags'

**Comment by hectcastro** [comment]
:+1:

**Pull request merged**